draft-ietf-quic-transport.md

```diff
@@ -819,14 +819,13 @@ frame.
 
 # Flow Control {#flow-control}
 
-It is necessary to limit the amount of data that a receiver could buffer, to
-prevent a fast sender from overwhelming a slow receiver, or to prevent a
-malicious sender from consuming a large amount of memory at a receiver.  To
-enable a receiver to limit memory commitment to a connection and to apply back
-pressure on the sender, streams are flow controlled both individually and as an
-aggregate.  A QUIC receiver controls the maximum amount of data the sender can
-send on a stream at any time, as described in {{data-flow-control}} and
-{{fc-credit}}.
+Receivers need to limit the amount of data that they are required to buffer, in
+order to prevent a fast sender from overwhelming them or a malicious sender from
+consuming a large amount of memory.  To enable a receiver to limit memory
+commitment to a connection and to apply back pressure on the sender, streams are
+flow controlled both individually and as an aggregate.  A QUIC receiver controls
```

Comment:
```suggestion
flow controlled both individually and across a connection.  A QUIC receiver controls
```

Reply:
Or
```suggestion
flow controlled both individually and across a connection as a whole.  A QUIC receiver controls
```

Reply:
I think what you might want is "as an aggregate spanning a connection", since the "connection" is describing the scope of aggregation, not the streams themselves. In the interest of keeping this less verbose, "as an aggregate" is probably enough. I'll take it as is, but if you feel strongly, I'm happy to do a separate commit for just this.